Sources/VaporJWT/Droplet+JWT.swift

```diff
@@ -0,0 +1,27 @@
+import Vapor
+import JWT
+
+extension Droplet {
+    /// Returns the main JWT signer
+    /// or throws an error if not properly configured
+    public func jwtSigner() throws -> Signer {
+        guard let signer = storage["jwtSigner"] as? Signer else {
+            throw ConfigError.noSigner()
+        }
+
+        return signer
+    }
+
+    /// Verifies the signature of a JWT
+    /// using the droplet's main JWT signer
+    public func verifySignature(jwt: JWT) throws {
+        let signer = try jwtSigner()
+        try jwt.verify(using: signer)
+    }
+
+    /// Used internally to set the droplet's
+    /// main JWT signer
+    internal func setJWTSigner(signer: Signer) {
```

Comment:
The `jwtSigner()` function can throw like the `req.auth.user()` one. I think this style is more appropriate than `Signer?` because the only time that this should return `nil` is if the project wasn't properly configured.

Totally agree on the naming thing. I actually already had changed that and haven't pushed yet haha.